### PR TITLE
ZCS-4919 : To download caldav / carddav configuration from the server

### PR DESCRIPTION
**Pull request by auminpatel**
Have created a new component under the Connected devices and Apps called as Mobile Configuration for the user to download to the caldav/carddav configuration from the server .
In this created a new template and in the view part have created a drop down for the selection of the type of configuration.

### CODE REVIEW

File: WebRoot/templates/prefs/Pages.template [hunk 1]
```diff
@@ -2091,7 +2091,48 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
...
+                       </tr>
+                        <tr><td colspan=2><hr></td></tr>
+                        <tr>
+                            <td>
+                                <div id='${id}_mobileConfig'></div>
+                            </td>
+                        </tr>
+                        <tr><td colspan=2><hr></td></tr>
+                        <tr>
+							<td>
```
Comment (silentsakky) [inline WebRoot/templates/prefs/Pages.template]: indentation

File: WebRoot/css/zm.css [hunk 2]
```diff
@@ -6274,3 +6274,13 @@ a.ZmLinkDisabled {
 	height: 100%;
 }
 /*Chat end */
+
+/*MObile Configuration start*/
+.ZmMobileConfig, .ZmMobileConfig:focus {
+    border: none !important;
```
[the remark this answers is not in the document]
Comment (auminpatel) [reply WebRoot/css/zm.css]: Have tried it without that but not possible 
have even tried using inline style but not possible

[the remark this answers is not in the document]
Comment (auminpatel) [reply WebRoot/css/zm.css]: Resolved

File: WebRoot/templates/prefs/Pages.template [hunk 3]
```diff
@@ -2091,7 +2091,46 @@ NOTE: The tables inside of the filter rule sections MUST contain a tbody element
...
+					<tr><td colspan=2><hr></td></tr>
+					<tr>
+						<td><div id='${id}_mobileConfig'></div></td>
+					</tr>
+					<tr><td colspan=2><hr></td></tr>
+					<tr>
+						<td>
+							<div><$=ZmMsg.mobileConfigurationUrls$></div>
+							<br/>
+							<div><textarea rows='1' cols='100%' class='ZmMobileConfigTextarea' readonly>&bull;&ensp;<$=AjxMessageFormat.format(ZmMsg.mobileConfigurationUrlCalDAV, [location.hostname, appCtxt.get(ZmSetting.USERNAME)])$></textarea></div>
```
Comment (silentsakky) [inline WebRoot/templates/prefs/Pages.template]: what is &bull; tag ?

Comment (auminpatel) [reply WebRoot/templates/prefs/Pages.template]: It is for used for bullets

**Pull request merged**